unirecfilter/lib/functions.c

```diff
@@ -72,6 +72,7 @@ struct ast *main_tree = NULL;
 extern struct yy_buffer_state * get_buf();
 extern struct yy_buffer_state * yy_scan_string(const char* yy_str);
 extern void yy_delete_buffer(struct yy_buffer_state * buffer);
+char * str_buffer;
```

Comment:
```suggestion
char *str_buffer;
```

Reply:
I tried to keep the style from "functions.h". You might want to make similar change there also.